src/entities/filters.ts

```diff
@@ -0,0 +1,29 @@
+export type CategoryKey =
+  | "ALL"
+  | "KOREAN"
+  | "CHINESE"
+  | "JAPANESE"
+  | "WESTERN"
+  | "ASIAN"
+  | "ETC";
+export const CATEGORY: Record<CategoryKey, string> = {
+  ALL: "전체",
+  KOREAN: "한식",
+  CHINESE: "중식",
+  JAPANESE: "일식",
+  WESTERN: "양식",
+  ASIAN: "아시안",
+  ETC: "기타",
+} as const;
```

Comment:
```suggestion
export type CategoryType =
  | "KOREAN"
  | "CHINESE"
  | "JAPANESE"
  | "WESTERN"
  | "ASIAN";
export const CATEGORY_META: Record<CategoryType | 'ALL' | 'ETC', string> = {
  ALL: "전체",
  KOREAN: "한식",
  CHINESE: "중식",
  JAPANESE: "일식",
  WESTERN: "양식",
  ASIAN: "아시안",
  ETC: "기타",
} as const;
```

- 여기서는 `CategoryType`을 써도 좋겠네요
- ALL, ETC는 `CategoryType`으로 만들 수 있는 정보로 보여 따로 선언하지 않아도 될 것 같습니다 !

Reply:
CategoryKey -> CategoryType 변경했습니다 
[8f06706](https://github.com/woowacourse/javascript-lunch/pull/236/commits/8f06706d3d00ade6489bb34ff13f81d8447fcf92)

다만, 'ETC'는 레스토랑 추가할 때 '기타'도 하나의 선택지이기 때문에 CategoryType으로 보는게 맞다고 판단했습니다! 
이외에 'ALL'이 필요한 부분에서는 말씀해주신 대로 CategoryType | "ALL" 로 만들어서 사용하도록 변경했습니다!